ui/src/components/ripple-out/RippleOutHeroSection.js

```diff
@@ -0,0 +1,179 @@
+import { Box, Container, Flex, Text, Center, HStack } from '@chakra-ui/react';
+import { useHistory } from 'react-router-dom';
+
+const RippleOutHeroSection = () => {
+  const history = useHistory();
+  const navStyle = {
```

Comment:
i dont think we ever use a separate CSS file tho? would it be better to keep it consistent and incld the styling in the files? esp since its only used in this file

Reply:
oh okok if only in this file, its fine. in some projects, we do keep separate css file like With Everything